billchop-fe/src/components/GroupPageHeader.tsx

```diff
@@ -1,58 +1,119 @@
 import * as React from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Col, Row } from "react-bootstrap";
 import ImageButton from "./ImageButton";
 import AddBillIcon from "../assets/add-bill-icon.svg";
+import AddPersonIcon from "../assets/add-person-icon.svg";
 import AddBillModal from "./AddBillModal";
+import SearchBox from "./SearchBox";
+import UserClient from "../backend/clients/UserClient";
+import User from "../backend/models/User";
 
 export interface IGroupPageHeaderProps {
+  groupId: string;
   onAddNewBill: (name: string, total: number) => void;
+  onAddNewMember: (userId: string) => void;
 }
 
 export interface IGroupPageHeaderState {
   showAddBillModal: boolean;
+  showSearchBox: boolean;
+  foundUsers: User[];
 }
 
 export default class GroupPageHeader extends React.Component<
   IGroupPageHeaderProps,
   IGroupPageHeaderState
-> {
+  > {
   constructor(props: IGroupPageHeaderProps) {
     super(props);
     this.state = {
       showAddBillModal: false,
+      showSearchBox: false,
+      foundUsers: [],
```

Comment:
[No Fix]
This and all search UI logic should probably be placed in its own sub-component at some point. Its fine for now, could be a refactoring task for next iteration.